.github/workflows/close-incomplete-issues.yml

```diff
@@ -17,3 +17,13 @@ jobs:
             If this has been closed in error, edit the title and body, and post a follow-up comment.
           title-contains: '<PUT TITLE HERE>'
           body-contains: "<!-- Tips: where applicable, specify browser name, browser version, and mobile operating system version -->\r\n\r\n#### What information was incorrect, unhelpful, or incomplete?\r\n\r\n#### What did you expect to see?\r\n\r\n#### Did you test this? If so, how?\r\n\r\n"
+      - uses: ddbeck/invalid-issue-closer@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          labels: 'invalid :no_entry_sign:'
+          comment: |
+            This issue was automatically closed because it appears to be empty. The issue is not different from the issue template and no actionable information has been provided.
+
+            If this has been closed in error, edit the title and body, and post a follow-up comment.
+          title-contains: '<SUMMARIZE THE PROBLEM>'
+          body-contains: "<!-- Tips: where applicable, specify browser name, browser version, and mobile operating system version -->\r\n\r\n#### What information was incorrect, unhelpful, or incomplete?\r\n#### What did you expect to see?\r\n#### Did you test this? If so, how?\r\n"
```

Comment:
ugh, what's up with GitHub and newlines? Did you have to fuss with newlines on your fork to get this to work?

Reply:
You guessed it, haha -- it took a bit of fiddling to get the newlines right!